### ISSUE TITLE
Enable 'Save image as...' on GitHub Pages

### ISSUE THREAD

**Issue by Hyperion2400** [cat-milk/Anime-Girls-Holding-Programming-Books#644]
Right now the component that holds the images has an overlay that prevents right-clicking and saving images.
Since the point of this repo is to share these images it would be nice if that could be fixed.

https://[cat-milk.github.io/Anime-Girls-Holding-Programming-Books/](https://cat-milk.github.io/Anime-Girls-Holding-Programming-Books/)

![image](https://github.com/cat-milk/Anime-Girls-Holding-Programming-Books/assets/34254020/263e980b-d0a1-41d4-b112-8c883cd8ecd2)

**Comment by THEGOLDENPRO** [comment]
yeah I see that too. Maybe we should consider #358 being the new site sometime. I allows the copying of images.

https://waifus-for-programmers.vercel.app/

**Comment by THEGOLDENPRO** [comment]
> yeah I see that too. Maybe we should consider #358 being the new site sometime. I allows the copying of images.
> 
> https://waifus-for-programmers.vercel.app/

Ah there's actually an issue on that #349
